kythe/cxx/indexer/cxx/testdata/proto/cc_proto_verifier_test.bzl

```diff
@@ -24,8 +24,7 @@ def cc_proto_verifier_test(
         name,
         srcs,
         proto_lib,
-        proto_srcs = [],
-	proto_deps = [],
+        proto_deps = [],
```

Comment:
should this not just be `deps`?

Reply:
(arguably, proto_lib should be a single-element-deps, similarly to how the language specific proto rules work)

Reply:
Removed `proto_deps` and `proto_lib` in favor of a single `proto_libs` argument. I adjusted the `proto_extract_kzip` rule to only list the top-level proto(s) of each lib as sources for the verifier, so some tests need to specify multiple proto libraries.